sema/check/testing/test_example.py

```
@@ -0,0 +1,42 @@
+# your_submodule/tests/test_example.py
+
+from .base import TestBase, TestResult
+
+
+class ExampleTest(TestBase):
+    def run(self) -> TestResult:
```

Ruff

Missing type annotation for `self` in method

(ANN101)